src/controllers/index.ts

```diff
@@ -1,2 +1,3 @@
 export { healthController } from './health.controller';
 export { notificationController } from './notification.controller';
+export { deviceController } from './device.controller';
```

Comment:
Please sort alphabetically. :slightly_smiling_face:

Reply:
I just corrected it and it shows like that in my local. Not sure why it is not showing up here. 
![image](https://user-images.githubusercontent.com/5863812/114761705-7a3bf580-9d26-11eb-8fbf-9cbd1151234a.png)